ovirt-ansible-repositories.spec.in

```diff
@@ -0,0 +1,54 @@
+%global rolename repositories
+%global roleprefix oVirt.
+%global roleprefix_legacy ovirt-
+%global ansible_roles_dir ansible/roles
+
+Name: @PACKAGE_NAME@
+Summary: Ansible role to create virtual machine infrastructure
+Version: @RPM_VERSION@
+Release: @RPM_RELEASE@%{?release_suffix}%{?dist}
+Source0: http://resources.ovirt.org/pub/src/@PACKAGE_NAME@/@PACKAGE_NAME@-@PACKAGE_VERSION@.tar.gz
+License: ASL 2.0
+Group:          Virtualization/Management
+BuildArch:      noarch
+Url:            http://www.ovirt.org
+
+Requires: ansible >= 2.4
+
+%description
+This Ansible role is used to set the repositories required for
+oVirt engine or host installation.
+
+%pretrans -p <lua>
```

Comment:
You can remove this `%pretrans`, it was used only for roles which existed in 4.1